Gemfile.lock

```diff
@@ -13,90 +13,99 @@ GEM
       execjs
     coffee-script-source (1.11.1)
     colorator (1.1.0)
-    commonmarker (0.17.9)
+    commonmarker (0.17.13)
       ruby-enum (~> 0.5)
-    concurrent-ruby (1.0.5)
-    ethon (0.11.0)
+    concurrent-ruby (1.1.5)
+    dnsruby (1.61.2)
+      addressable (~> 2.5)
+    em-websocket (0.5.1)
+      eventmachine (>= 0.12.9)
+      http_parser.rb (~> 0.6.0)
+    ethon (0.12.0)
       ffi (>= 1.3.0)
+    eventmachine (1.2.7)
     execjs (2.7.0)
-    faraday (0.14.0)
+    faraday (0.15.4)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.23)
+    ffi (1.10.0)
     forwardable-extended (2.6.0)
     gemoji (3.0.0)
-    github-pages (179)
-      activesupport (= 4.2.9)
-      github-pages-health-check (= 1.4.0)
-      jekyll (= 3.6.2)
-      jekyll-avatar (= 0.5.0)
+    github-pages (197)
+      activesupport (= 4.2.10)
```

Comment:
Is there a higher version we can go to here? 4.2 is quite old.

Reply:
It's a hard dependency by `github-pages` so far https://rubygems.org/gems/github-pages